metadata.rb

```diff
@@ -22,9 +22,9 @@
 license          'Apache-2.0'
 description      'Installs and configures operating system hardening'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '3.1.0'
+version          '4.0.0'
```

Comment:
can you please remove the version bump, it will be done by us during the release. Thanks!